runtime/hydradx/Cargo.toml

```diff
@@ -51,7 +51,7 @@ hydradx-traits = { git = "https://github.com/galacticcouncil/warehouse", rev = "
 pallet-relaychain-info = { git = "https://github.com/galacticcouncil/warehouse", rev = "8b6bc9f287addba4ac1e4786118edc5154464d78", default-features = false }
 pallet-transaction-multi-payment = { git = "https://github.com/galacticcouncil/warehouse", rev = "8b6bc9f287addba4ac1e4786118edc5154464d78", default-features = false}
 pallet-asset-registry = { git = "https://github.com/galacticcouncil/warehouse", rev = "8b6bc9f287addba4ac1e4786118edc5154464d78", default-features = false}
-pallet-collator-rewards = { git = "https://github.com/galacticcouncil/warehouse", rev = "08a9b3f8e7c690ce8282e0222d1e594988e58831", default-features = false}
+pallet-collator-rewards = { git = "https://github.com/galacticcouncil/warehouse", branch = "apopiak/collator-rewards", default-features = false}
```

Comment:
This can lead to different warehouse pallets being included which then in turn could pull in different versions of substrate, no?

Reply:
not as long that branch stays on the same substrate, we can pin it on specific rev when the warehouse pr is merged